src/progressbar/progressbar.ts

```diff
@@ -14,10 +17,30 @@ import {getValueInRange, toBoolean} from '../util/util';
   `
 })
 export class NgbProgressbar {
+  /**
+   * Maximal value to be displayed in the progressbar.
+   */
   @Input() max: number = 100;
+
+  /**
+   * A flag indicating if a progress bar should be animated when the value changes. Takes effect only for browsers
```

Comment:
Is this note needed about CSS3 animations? I feel like all the browsers we're supporting support them.